cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/catalog/CloudbreakVersion.java

```diff
@@ -13,20 +13,28 @@
 
     private final List<String> versions;
 
+    private final List<String> defaults;
+
     private final List<String> imageIds;
 
     @JsonCreator
     public CloudbreakVersion(
             @JsonProperty("versions") List<String> versions,
+            @JsonProperty("defaults") List<String> defaults,
```

Comment:
deafults enough or should it be default-images ?

Reply:
this is coming from the image-catalog

Reply:
I see but then should we use a different naming there ?

Reply:
No, it fits the image catalog schema and we agreed on this naming.

Reply:
ok